src/app/pages/NewWallet/setWalletPassword/SetWalletPassword.tsx

```diff
@@ -295,7 +295,7 @@ export const SetWalletPassword: FC<SetWalletPasswordProps> = ({
       <FormSubmitButton
         loading={submitting}
         style={{ display: 'block', width: '100%', fontSize: 14, fontWeight: 500 }}
-        testID={setWalletPasswordTestIDS.importButton}
+        testID={setWalletPasswordTestIDS.completeButton}
```

Comment:
use 2 different selectors, as it works for text bellow `<T id={ownMnemonic ? 'import' : 'create'} />`